package/src/components/Channel/Channel.tsx

```diff
@@ -582,6 +583,7 @@ const ChannelWithContext = <
     setTyping,
     setWatcherCount,
     setWatchers,
+    shouldShowUnreadUnderlay = true,
```

Comment:
I'm now thinking about whether it would be better to introduce a theme color property specifically for the underlay and just make it default to `bg_gradient_start`, or better yet introduce an override for the entire container within the `theme`. What do you think @khushal87 ?

Reply:
Yep, good idea. I will add it

Reply:
Done